test_util/testutil.cc

```diff
@@ -259,6 +259,26 @@ TableFactory* RandomTableFactory(Random* rnd, int pre_defined) {
 #endif  // !ROCKSDB_LITE
 }
 
+class UnregisteredTableFactoryImpl : public TableFactory {
+ public:
```

Comment:
Is there a reason not to put this class only in options_test?  Then the class can be local to there and not need the constructor in

Reply:
Done.